web/Web/web/CAL/models.py

```diff
@@ -0,0 +1,38 @@
+from config.settings.base import AUTH_USER_MODEL as User
+
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from web.core.models import Session
+
+
+class DS_logging(models.Model):
```

Comment:
[Model names should be singular](https://simpleisbetterthancomplex.com/tips/2018/02/10/django-tip-22-designing-better-models.html#naming-your-models). Let's rename it to `Stratum`

Reply:
Sure!